File: api_gateway/src/main/java/com/ticketflow/api_gateway/models/ticket_service/OrderModel.java
package com.ticketflow.api_gateway.models.ticket_service;

public class OrderModel {
    private Integer ticketId;
    private String buyerEmail;

    public OrderModel(Integer ticketId, String buyerEmail) {
        this.ticketId = ticketId;
        this.buyerEmail = buyerEmail;
    }

    public Integer getTicketId() {
        return ticketId;
    }

    public String getBuyerEmail() {
        return buyerEmail;
    }
}

File: backend/dotnet/ticketflow/TicketFlow.TicketService.Client/Extensibility/Models/OrderModel.cs
﻿namespace TicketFlow.TicketService.Client.Extensibility.Models
{
    public class OrderModel
    {
        public OrderModel(int ticketId, string buyerEmail)
        {
            TicketId = ticketId;
            BuyerEmail = buyerEmail;
        }

        public int TicketId { get; }

        public string BuyerEmail { get; }
    }
}